tutorials/selenium/README.md

```diff
@@ -1,28 +1,68 @@
 
-This tutorial demonstrates how to create a simple Selenium WebDriver application that is configured
-to access a web page loaded in a desktop Java app using JxBrowser on Windows.
+This tutorial demonstrates how to create a simple Selenium WebDriver application
+that is configured to access a web page in a desktop Java application based on
+JxBrowser.
 
 Creating of these example projects is described in
 the [tutorial](https://jxbrowser-support.teamdev.com/docs/tutorials/integration/selenium.html).
 
+The tutorial is available on all supported platforms.
+
+## Prerequisites
+
+The tutorial requires the `jpackage` tool available in `PATH` so make sure that
+the environment variable contains the `%JAVA_HOME%/bin` directory.
+
 ## Building and Launching
 
-1. Download [ChromeDriver](https://sites.google.com/a/chromium.org/chromedriver/downloads). Put it
-   into the `launcher/src/main/resources` directory. Use one of the latest versions.
-2. [Insert your license key](https://jxbrowser-support.teamdev.com/docs/guides/licensing.html#adding-the-license-to-a-project)
-   into [the test application](https://github.com/TeamDev-IP/JxBrowser-Examples/blob/90fdd92f7c4c8737929f57e0383aad39d4be2aee/tutorials/selenium/target-app/src/main/java/TargetApp.java#L45)
-   .
-3. In the [target-app](target-app) module run the `createExe` Gradle task.
-4. Run [Selenium](launcher/src/main/java/SeleniumLauncher.java).
+1. In the [root](../..) directory, run the `downloadChromeDriver` task of
+   the [launcher](launcher) module to download ChromeDriver to the
+   `resources` directory:
+   
+   **Windows:**
+   ```bash
+   gradlew launcher:downloadChromeDriver
+   ```
+   
+   **macOS/Linux:**
+     ```bash
+   ./gradlew launcher:downloadChromeDriver
+   ```
 
-   _You may need to run Selenium twice as during the first start ChromeDriver may not detect the
-   running application binaries._
+   > **_NOTE:_** Each version of ChromeDriver corresponds to the same version of
+   Chromium used in JxBrowser. So if you change the JxBrowser version in the
+   root [build.gradle.kts](../../build.gradle.kts) you need to update the
+   `chromiumVersion` property [here](launcher/build.gradle.kts)
+   before running the `downloadChromeDriver` task. Otherwise, Selenium WebDriver
+   will not be able to connect to Chromium. You can find the Chromium version of
+   the specific JxBrowser version in
+   the [release notes](https://teamdev.com/jxbrowser/release-notes/) of the
+   last, or in one of the previous version where Chromium has been updated.
+2. [Insert your license key](https://teamdev.com/jxbrowser/docs/guides/introduction/licensing/#adding-the-license-to-a-project)
+   into [the test application](./target-app/src/main/java/TargetApp.java).
+3. In the [root](../..) directory, run the `buildApplication` task
+   of the [target-app](target-app) module to create an executable file for the
+   current platform that later will be run by Selenium WebDriver.
```

Comment:
```suggestion
   Each version of ChromeDriver corresponds to the Chromium version used in JxBrowser. If you change the JxBrowser version in [build.gradle.kts](../../build.gradle.kts) you need to update the
   `chromiumVersion` property [here](launcher/build.gradle.kts)
   before running the `downloadChromeDriver` task. Otherwise, Selenium WebDriver
   will not be able to connect to Chromium. You can find which Chromium version is used in the specific JxBrowser version in
   the [release notes](https://teamdev.com/jxbrowser/release-notes/).
2. [Insert your license key](https://teamdev.com/jxbrowser/docs/guides/introduction/licensing/#adding-the-license-to-a-project)
   into [the test application](./target-app/src/main/java/TargetApp.java).
3. In the [root](../..) directory, run the `buildApplication` task
   of the [target-app](target-app) module to create an executable file for the
   current platform that later will be run by Selenium WebDriver.
```